nni/nas/oneshot/pytorch/strategy.py

```diff
@@ -159,5 +159,53 @@ def __init__(self, **kwargs):
         super().__init__(RandomSamplingLightningModule, **kwargs)
 
     def sub_state_dict(self, arch: dict[str, Any]):
+        """Export the state dict of a chosen architecture.
+        This is useful in weight inheritance of subnet as was done in
+        `SPOS <https://arxiv.org/abs/1904.00420>`__,
+        `OFA <https://arxiv.org/abs/1908.09791>`__ and
+        `AutoFormer <https://arxiv.org/abs/2106.13008>`__.
+
+        Parameters
+        ----------
+        arch
+            The architecture to be exported.
+
+        Examples
+        --------
+        To obtain a state dict of a chosen architecture, you can use the following code::
+
+            # Train or load a random one-shot strategy
+            experiment.run(...)
+            best_arch = experiment.export_top_models()[0]
+
+            from nni.nas.fixed import no_fixed_arch
+            with no_fixed_arch():               # use this in evaluator, as evaluator is running in a fixed context.
```

Comment:
the comment in this line is confusing. do you mean "If users load subnet's checkpoint in an evaluator, they should use this `with no_fixed_arch()` statement to make sure instantiating model space works correctly, because evaluator is running in a fixed context"?

Reply:
Sounds more clear.